.travis.yml

```diff
@@ -22,9 +27,12 @@ compiler:
   - gcc
   - clang
 
+python:
+  - '2.7'
+
 env:
   global:
-    - CMAKE_FLAGS="-DWITH_SPOTLESS:BOOL=OFF"
+    - CMAKE_FLAGS="-DWITH_SPOTLESS:BOOL=OFF -DWITH_LIBBOT:BOOL=ON -DWITH_GTK:BOOL=ON -DWITH_DIRECTOR:BOOL=ON -DWITH_IRIS:BOOL=ON-DWITH_OCTOMAP:BOOL=ON -DWITH_MOSEK:BOOL=ON -DWITH_AVL:BOOL=ON -DWITH_XFOIL:BOOL=ON"
```

Comment:
Some of the supported externals require matlab but don't check whether it
exists yet.

On Wed, Aug 19, 2015 at 5:02 PM, Russ Tedrake <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/RobotLocomotion/drake/pull/1254#discussion_r37467602>:
>
> >  env:
> >    global:
> > -    - CMAKE_FLAGS="-DWITH_SPOTLESS:BOOL=OFF"
> > +    - CMAKE_FLAGS="-DWITH_SPOTLESS:BOOL=OFF -DWITH_LIBBOT:BOOL=ON -DWITH_GTK:BOOL=ON -DWITH_DIRECTOR:BOOL=ON -DWITH_IRIS:BOOL=ON-DWITH_OCTOMAP:BOOL=ON -DWITH_MOSEK:BOOL=ON -DWITH_AVL:BOOL=ON -DWITH_XFOIL:BOOL=ON"
>
> why not -DWITH_ALL_SUPPORTED_EXTERNALS=ON ? it's more likely to stay up to
> date.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RobotLocomotion/drake/pull/1254/files#r37467602>.
>



-- 
Robin L. H. Deits
Robot Locomotion Group @CSAIL
Massachusetts Institute of Technology